src/components/Bar.jsx

```diff
@@ -193,7 +195,14 @@ export const Bar = ({
   }
 
   return (
-    <div className="coz-bar-wrapper" data-testid="coz-bar-wrapper">
+    <div
+      {...componentsProps?.CozBarWrapper}
```

Comment:
we are in CozBar, maybe `componentsProps?.Wrapper` is enough?

Reply:
Deal. Don't like "CozBarWrapper" too.